slurp/src/main/webapp/index.html

```diff
@@ -2,17 +2,21 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>Slurp</title>
+    <!-- CSS -->
+    <link rel="stylesheet" href="/styles/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="styles/style.css">
```

These links should be relative.

Reply:
Sure. I am curious why though since they have the same effect?